source/_patterns/01-molecules/content-meta/content-meta.twig

```diff
@@ -19,15 +17,20 @@
               {%- endif -%}
 
               {%- include 'atoms-link' -%}
-
             {%- endblock content -%}
           {%- endwith -%}
 
         </li>
 
       {%- endblock item -%}
+
     {%- endwith -%}
 
   {%- endfor -%}
 
+  {%- if date is defined -%}
+    {%- set attributes = attributes|default({})|merge({class: []|merge(['content-meta__item']) }) -%}
+    <li {%- include('atoms-attributes') -%}>{%- include 'atoms-date' -%}</li>
```

Comment:
This needs to be a normal item in `items`, so this pattern won't contain a direct `include` to `atoms-date`, instead the data given to it does, like https://github.com/libero/pattern-library/blob/63cea902e49662df2e6e12ebe2167aaa28b1128f/source/_patterns/00-atoms/block/paragraph/paragraph.yaml#L3-L5

We could update https://github.com/libero/pattern-library/blob/63cea902e49662df2e6e12ebe2167aaa28b1128f/source/_patterns/01-molecules/content-meta/content-meta.twig#L15-L19 to accept one of these directly, rather than having to wrap it in `text` too.